Included redux logger

elizabetdev opened the pull request.

elizabetdev committed:
Included redux logger

punkbit reviewed (changes requested):
Looks good, can you remove the `/dist/index.html` from the repository ?

elizabetdev committed:
Ignore dist folder

elizabetdev committed:
Removing dist files in .gitignore

punkbit committed:
Update .gitignore

the dist directory does not need to be in the ignore list, cases like herokuapp or azures server may require the dist directory in the project, and this may cause initial confusion in why not able to track
